app/modules/api/batch.js

```diff
@@ -7,7 +7,7 @@ import {
 } from '../../values/api'
 
 type State = {
-  batch: true,
+  batch: boolean,
```

Comment:
I had this as `true` because a state should never come throw here with `batch: false` FWIW.